### PR TITLE
Nips 2017

### PR DESCRIPTION
**Pull request by pmandler**
Forgot to name it feature.
This is the code for the two new NIPS 2017 Workshops, MLITS and Autodiff.

### CODE REVIEW

File: venues/NIPS.cc/2017/Workshop/Autodiff/params.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (mspector) [inline venues/NIPS.cc/2017/Workshop/Autodiff/params.json]: Do you need this file and the file /data/NIPS2017_autodiff.json ? they do look different but they look like they're achieving similar things...

File: venues/NIPS.cc/2017/Workshop/MLITS/params.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (mspector) [inline venues/NIPS.cc/2017/Workshop/MLITS/params.json]: same here. what's the difference between these two json files?

File: venues/NIPS.cc/2017/Workshop/Autodiff/params.json (hunk 1 above)
Comment (pmandler) [reply venues/NIPS.cc/2017/Workshop/Autodiff/params.json]: They are the files used by admin/superuser_init.py to create the conferences.  So yes they are very similar, but different titles and due dates.

Comment (mspector) [reply venues/NIPS.cc/2017/Workshop/Autodiff/params.json]: I still don't fully understand why we need both of them. The `conference_title`s look the same to me... am I missing something?

File: venues/NIPS.cc/2017/Workshop/MLITS/params.json (hunk 2 above)
Comment (pmandler) [reply venues/NIPS.cc/2017/Workshop/MLITS/params.json]: One says 
"MLITS Workshop" - "Machine Learning for Intelligent Transportation Systems", and is due on Oct 12 the other says 
"Autodiff Workshop" - "The future of gradient-based machine learning software and techniques" and is due on Oct 14.

Comment (mspector) [reply venues/NIPS.cc/2017/Workshop/MLITS/params.json]: Yes, I understand that there are two different conferences, but there are four total files, two of which seem redundant:

venues/NIPS.cc/2017/Workshop/MLITS/params.json seems redundant with venues/NIPS.cc/2017/Workshop/MLITS/data/NIPS2017_MLITS.json,

and 

venues/NIPS.cc/2017/Workshop/Autodiff/params.json seems redundant with
venues/NIPS.cc/2017/Workshop/Autodiff/data/NIPS2017_autodiff.json

**Comment by melisabok** [comment]
What about this? should we merge it? is this code already in live?

**Comment by pmandler** [comment]
Yes it should be merged.  And yes it is already live (oops). 
I tried to remove the redundant files NIPS2017_MLITS.json and the autodiff version. They are gone on my copy, but I’m not sure I got them out of this check in.

Pam
___________________
Pam Mandler
Center for Data Science
 <https://ds.cs.umass.edu/>(413) 545-2561

**Pull request merged**